### PR TITLE
fix(progress): remove minimum width variable

### LINKED ISSUE #913: PF4 progress bar shows 10% as min value
The Koku UI currently uses the latest PF4 progress bar and in some cases, shows zero as the percentage value. However, the progress bar itself still shows zero as 10%.

Similar to how 100% shows a full bar, I expect zero to show an empty bar.

Zero Percent
image

10 Percent
image

### PR DESCRIPTION
**Pull request by ibolton336**
fixes #913 

Original issue in patternfly-react: https://github.com/patternfly/patternfly-react/issues/888

### CODE REVIEW

File: src/patternfly/components/Progress/progress.scss [hunk 1]
```diff
@@ -18,7 +18,7 @@
   --pf-c-progress__bar--Height--sm: var(--pf-global--spacer--sm);
   --pf-c-progress__bar--Height--lg: var(--pf-global--spacer--lg);
   --pf-c-progress__bar-overlay--Opacity: .2; // one-off change in opacity to allow progress bar to automatically coordinate with the indicator color
-  --pf-c-progress__indicator--MinWidth: var(--pf-global--spacer--xl);
+  --pf-c-progress__indicator--MinWidth: 0;
```
Comment (matthewcarleton) [inline src/patternfly/components/Progress/progress.scss]: you can just ditch this variable if you don't want to pull it in on line 141. I don't think we need to set them for 0.

Comment (ibolton336) [reply src/patternfly/components/Progress/progress.scss]: Ah that makes sense. Updating now.

**Comment by matthewcarleton** [comment]
@ibolton336 just wanted to check in on the status of this. If you are blocked just let us know and we can help :)

**Comment by ibolton336** [comment]
> That min-width was there so that when the measure is shown inside the bar, there is enough room for it. I had talked with either @mcarrano or @mceledonia about that, I don't remember which. So probably a better fix would be to only have a min-width for that option instead of all.

Do you think it would be fair to ask the end user to create a min width if they wish (when using the inline label ) ?  That seems to be what react-bootstrap does ? https://react-bootstrap.github.io/components/progress/  

@srambach

**Comment by srambach** [comment]
The semantic example you had https://semantic-ui.com/modules/progress.html seems to be consistent with how ours currently is, with a min-width.
My recollection is that we wanted to make sure that the inside measure fit in the bar, and that the indication of 0% was enough to allow the user to understand where progress stood. The min-width was sized to make sure the text would fit without half overlapping, which was determined to be a problem. Allowing the user to specify this could result in this undesirable display condition.
So, my inclination would be to leave the min-width as is on the inside measure. If design/interaction (@mceledonia @mcarrano ) feel that it's better to remove the min-width from the outside or no measure versions, then it should be easy just to apply the min-width to the inside measure version.

**Comment by mcarrano** [comment]
I agree that it looks like a bug when the value displays 0% but the bar indicates something different. @srambach what would it take to update this so the min-width only applies when the value is reported inside of the bar?

**Comment by srambach** [comment]
@mcarrano The CSS for that shouldn't be hard. I can take a look if you want me to, @ibolton336

**Comment by srambach** [comment]
This PR https://github.com/patternfly/patternfly-next/pull/940 removes the min-width except in the case of showing the percentage within the progress bar.

**Pull request closed without merging**